src/it/java/se/knowit/bookitevent/ClientUtilIT.java

```diff
@@ -80,9 +70,9 @@ public void testCreateNewEvent() throws JsonProcessingException {
 		String jsonEvent = "{\"eventId\":\"1a0b7c8b-c0d5-4ab2-8c63-5cf1ad0b439b\",\n" + 
 				"                \"name\":\"IT event\",\n" + 
 				"                \"description\":\"IT description\" , \n" + 
-				"                \"eventStart\":\"2019-12-02T18:00:00\", \n" + 
-				"                \"eventEnd\":\"2019-12-02T22:00:00\", \n" + 
-				"                \"deadlineRVSP\":\"2019-11-02T17:00:00\", \n" + 
+				"                \"eventStart\":\"90000000\", \n" + 
+				"                \"eventEnd\":\"90060000\", \n" + 
```

Comment:
Not sure if it makes any big difference but we can skip the citation marks around the numerical values.
So \"eventStart\":90000000, instead of \"eventStart\":\"90000000\",